src/connect.js

```diff
@@ -24,7 +24,9 @@
  * @return {Function}
  */
 function connect(baseParams) {
-  // write code here
+  return (extraParams) => {
+    return Object.assign({}, { ...baseParams }, { ...extraParams });
```

Comment:
Можно просто 
```suggestion
    return { ...baseParams, ...extraParams };
```
Результат одинаковый

Reply:
I knew it